pkg/sentry/platform/ring0/lib_arm64.go

```diff
@@ -16,10 +16,27 @@
 
 package ring0
 
-// LoadFloatingPoint loads floating point state by the most efficient mechanism
-// available (set by Init).
-var LoadFloatingPoint func(*byte)
+// CPACR_EL1, Architectural Feature Access Control Register.
+// Get the value of CPACR_EL1.
+func GetCpacrEl1() (value uintptr)
 
-// SaveFloatingPoint saves floating point state by the most efficient mechanism
-// available (set by Init).
-var SaveFloatingPoint func(*byte)
+// FPCR, Floating-point Control Register.
+// Get the vaule of FPCR.
+func GetFpcr() (value uintptr)
+
+// Set the value of FPCR.
+func SetFpcr(value uintptr)
```

Comment:
These are all initialisms, so they should be SetFPCR, etc.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done.